### PR TITLE
pybind/mgr/dashboard: bind to :: by default

### PR DESCRIPTION
**Pull request by liewegas**
This is all available v4 and v6 addrs on the host.

Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

**Comment by liewegas** [comment]
Since we now have an explict 'ceph mgr module enable dashboard' step it is safe to default to bind to :: (all addresses on the host).  Combined with the default port of 7000 means we have a single command to turn on the dashboard.

File: src/pybind/mgr/dashboard/module.py [hunk 1]
```diff
@@ -744,7 +744,7 @@ def mds_counters(self, fs_id):
 
                 return dict(result)
 
-        server_addr = self.get_localized_config('server_addr')
+        server_addr = self.get_localized_config('server_addr') or '::'
```
Comment (tchaikov) [inline src/pybind/mgr/dashboard/module.py]: could just put

```py
server_addr = self.get_localized_config('server_addr', '::')
```

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-07-09_19:14:46-rados-wip-sage-testing-distro-basic-smithi/

**Comment by jcsp** [comment]
This probably makes Wido's recent doc changes out of date again (the part about where it listens by default was removed)

**Comment by liewegas** [comment]
see https://github.com/ceph/ceph/pull/16241

**Pull request merged**